Core/HLE/HLE.cpp

```diff
@@ -626,7 +626,7 @@ size_t hleFormatLogArgs(char *message, size_t sz, const char *argmask) {
 			break;
 
 		case 'i':
-			APPEND_FMT("%d", regval);
+			APPEND_FMT("%u", regval);
```

Comment:
Hi, well searching, with yours idea of use union, seems to remove this type of line:
```u32 *rdu = (u32 *)rd;```
its necesary use ```ReadVector``` with ```FLoatBits``` like:
```void ReadVector(FloatBits *rd, VectorSize size, int reg)```
this is only because ```ReadVector``` use floats and u32.
what do you think?